### PR TITLE
feat/ add secondary sidebar to object root page

### PR DESCRIPTION
**Pull request by BelgianNoise**

### CODE REVIEW

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 1]
```diff
@@ -160,11 +170,37 @@ export class ObjectRootComponent extends RxLitElement {
...
       <div slot="actions"><button class="no-padding inverse delete" @click="${() => this.actor.send(ObjectEvents.CLICKED_DELETE, { object: this.object })}">${unsafeSVG(Trash)}</button></div>
     </nde-content-header>
+    <div class="content-and-sidebar">
+
+    <nde-sidebar>
+      <nde-sidebar-item>
+        <div slot="content">
+          Een week geleden aangepast door Pieter Pieters
+        </div>
+      </nde-sidebar-item>
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:180]: I'd leave this out for now, as we can't fill it in with anything useful. I think @wouteraj said the same last week.

File: packages/solid-crs-manage/lib/features/object/object-root.component.ts [hunk 2]
```diff
@@ -160,11 +170,37 @@ export class ObjectRootComponent extends RxLitElement {
...
+      <nde-sidebar-item .padding="${false}" .showBorder="${false}">
+        <nde-sidebar-list slot="content">
+          <nde-sidebar-list-item slot="item" isTitle>
```
Comment (lem-onade) [inline packages/solid-crs-manage/lib/features/object/object-root.component.ts:183]:
```suggestion
          <nde-sidebar-list-item slot="title" isTitle>
```

To align with the primary sidebar's title

Comment (BelgianNoise) [reply packages/solid-crs-manage/lib/features/object/object-root.component.ts:183]: `slot=title` does not allign with figma

File: packages/solid-crs-manage/package.json [hunk 3]
```diff
@@ -82,10 +82,10 @@
     ],
     "coverageThreshold": {
       "global": {
-        "statements": 88.9,
+        "statements": 88.81,
```
Comment (lem-onade) [inline packages/solid-crs-manage/package.json:85]: 👀

**Review by lem-onade — APPROVED** [review]

**Pull request merged**